ubuntudesign/vanilla-framework
# Change Ubuntu orange

## Done

Change ubuntu orange to the new colour

## QA

Run ‘gulp build’ in the appropriate place and make sure the oranges are #e95420 and not #dd4814 in the demo

## Details

Card: https://trello.com/c/1Z3V717W
